yarn.lock

```diff
@@ -4661,6 +4643,116 @@
     regenerator-runtime "^0.13.7"
     resolve-from "^5.0.0"
 
+"@swc-node/core@^1.9.0":
```

Comment:
are we obliged to take all the swc deps ?

Reply:
it's a dependency of nx and we will start using it